src/main/java/frc/trigon/robot/subsystems/turret/toohardturret/ToohardTurretIO.java

```diff
@@ -0,0 +1,29 @@
+package frc.trigon.robot.subsystems.turret.toohardturret;
+
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.trigon.robot.subsystems.turret.TurretIO;
+import frc.trigon.robot.subsystems.turret.TurretInputsAutoLogged;
+
+public class ToohardTurretIO extends TurretIO {
+    private final TalonFX motor = ToohardTurretConstants.MOTOR;
+    private final MotionMagicVoltage positionRequest = new MotionMagicVoltage(0).withEnableFOC(ToohardTurretConstants.IS_FOC_ENABLED);
+
+    @Override
+    protected void updateInputs(TurretInputsAutoLogged inputs) {
+        inputs.motorVoltage = motor.getMotorVoltage().refresh().getValue();
```

Comment:
You're not using status signals correctly.

Reply:
Do you mean make a status signal?

Reply:
Yes